docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -325,53 +325,88 @@ The setting relies on the modification time of the file to determine if a file i
 
 To remove the state of previously harvested files from the registry file, use the `clean_inactive` configuration option.
 
-## Take over [filebeat-input-filestream-take-over]
+#### `take_over` [filebeat-input-filestream-take-over]
+
+```{applies_to}
+stack: beta
+```
+
+::::{warning}
+The take over mode is still in beta, however, it should be generally safe to use.
+::::
+
 When `take_over` is enabled, this `filestream` input will take over
 states from the [`log`](/reference/filebeat/filebeat-input-log.md) input
 or other `filestream` inputs. Only states of files being actively
 harvested by this input are taken over.
 
-To take over files from a `log` input, simply set `take_over.enabled: true`.
+The syntax for enabling take over mode varies by version:
+
+* {applies_to}`stack: beta 9.0.0` Use `take_over: true`.
+* {applies_to}`stack: beta 9.1.0` Use `take_over.enabled: true`.
+
+:::{note}
+While `take_over: true` is still supported to migrate state from the `log` input to
+`filestream`, we plan to remove support in a future version so you should use
+the new syntax if possible.
+:::
+
+##### `log` input
+
+To take over files from a `log` input, enable take over mode
+and make sure the files you want this input to take over match the configured globs in `paths`.
+
+::::{tab-set}
+:::{tab-item} 9.1.0
+```yaml
+take_over:
+  enabled: true
+```
+:::
+:::{tab-item} 9.0.0
+```yaml
+take_over: true
+```
+:::
+::::
+
 
-To take over states from other `filestream` inputs, set
-`take_over.enabled: true` and set `take_over.from_ids` to a list of
-existing `filestream` IDs you want to migrate files from.
 
-On both cases make sure the files you want this input to take over
-match the configured globs in `paths`.
+##### Other `filestream` inputs
+
+```{applies_to}
+stack: beta 9.1.0
+```
+
+To take over states from other `filestream` inputs, enable take over mode,
+set `take_over.from_ids` to a list of existing `filestream` IDs you want to migrate files from,
+and make sure the files you want this input to take over match the configured globs in `paths`.
 
 When `take_over.from_ids` is set, files are not taken over from `log`
 inputs. The migration is limited to `filestream` inputs only.
 
 ```yaml
 take_over:
   enabled: true
-  from_ids: ["foo", "bar"] # omit to take over from the log input
+  from_ids: ["foo", "bar"]
 ```
+
 :::{important}
-The `take over` mode can work correctly only if the source (taken from) inputs are no longer active. If source inputs are still harvesting the files which are being migrated, it will lead to data duplication and in some cases might cause data loss.
+The take over mode can work correctly only if the source (taken from) inputs are no longer active. If source inputs are still harvesting the files which are being migrated, it will lead to data duplication and in some cases might cause data loss.
```

Comment:
Both "important" blocks apply to the take over feature in general, it does not matter if the states are being taken over from the `log` or `filestream` input.

Having them under the "Other `filestream` inputs" is a bit confusing to me.

I also don't think having that many warnings/boxes at the very beginning of the description is good... Right now I don't really have a good suggestion on how to improve it :/

Reply:
What do you think about this approach? https://docs-v3-preview.elastic.dev/elastic/beats/pull/45735/reference/filebeat/filebeat-input-filestream#notes-and-limitations